src/main/java/com/edmunds/tools/databricks/maven/util/SettingsUtils.java

```diff
@@ -0,0 +1,169 @@
+/*
+ *  Copyright 2019 Edmunds.com, Inc.
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License");
+ *      you may not use this file except in compliance with the License.
+ *      You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *      Unless required by applicable law or agreed to in writing, software
+ *      distributed under the License is distributed on an "AS IS" BASIS,
+ *      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *      See the License for the specific language governing permissions and
+ *      limitations under the License.
+ */
+
+package com.edmunds.tools.databricks.maven.util;
+
+import com.edmunds.tools.databricks.maven.BaseDatabricksMojo;
+import com.edmunds.tools.databricks.maven.model.BaseModel;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import freemarker.cache.FileTemplateLoader;
+import freemarker.cache.StringTemplateLoader;
+import freemarker.cache.TemplateLoader;
+import freemarker.template.Configuration;
+import freemarker.template.Template;
+import freemarker.template.TemplateException;
+import freemarker.template.TemplateExceptionHandler;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.logging.Log;
+import org.apache.maven.plugin.logging.SystemStreamLog;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.nio.charset.Charset;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @param <M> Databricks Mojo
+ * @param <T> Template Model
+ * @param <D> Settings DTO
+ */
+public class SettingsUtils<M extends BaseDatabricksMojo, T extends BaseModel, D> {
```

Comment:
is there a better name for it then SettingsUtils? What is really the purpose of this class?

Please add this to your javadoc string too

Reply:
Done